### ISSUE TITLE
Update Reference to Grafana.net to Grafana.com

### ISSUE THREAD

**Issue by bulletfactory** [grafana/grafana#8415]
Read before posting: 

- Questions should be posted to https://community.grafana.com. Please search there and here on GitHub for similar issues before creating a new issue. 
- Checkout FAQ: https://community.grafana.com/c/howto/faq
- Checkout How to troubleshoot metric query issues: https://community.grafana.com/t/how-to-troubleshoot-metric-query-issues/50

Please prefix your title with [Bug] or [Feature request]. 

Please include this information:
- What Grafana version are you using?
- What datasource are you using?
- What OS are you running grafana on?
- What did you do?
   When logging in to HG there it still says log in w/ Grafana.net
- What was the expected result?
   Should say Grafana.com
- What happened instead?
- If related to metric query / data viz:
  - Include raw network request & response: get by opening Chrome Dev Tools (F12, Ctrl+Shift+I on windows, Cmd+Opt+I on Mac), go the network tab.
 
<img width="861" alt="screenshot 2017-05-18 10 44 44" src="https://cloud.githubusercontent.com/assets/1950595/26208101/4706aedc-3bb7-11e7-8f4d-338f210cd12e.png">

**Comment by mattttt** [comment]
@bulletfactory: Do you have the updated graphic?

**Comment by torkelo** [comment]
made something simple with grafana swirl and text
![image](https://cloud.githubusercontent.com/assets/10999/26309332/e6e8b754-3efc-11e7-8c14-201ab1130e35.png)

Looks like this if login form is disabled:

![image](https://cloud.githubusercontent.com/assets/10999/26309540/bb35189a-3efd-11e7-8de6-7ec63fe35e88.png)

removed With in the button as it is already included in the section heading "Or login with", but that heading was not shown when login form was disabled so added it back (but removed Or)

**Comment by DanCech** [comment]
@torkelo the url change in c34db77f042569fd596900e127c1322f9c7382ac completely breaks any existing oauth login via grafana.com, and needs to be reverted ASAP.

**Comment by DanCech** [comment]
Thinking some more, we may be able to support it via workaround in the grafana.com oauth code if we really want to update the url, let's chat tomorrow and figure out a plan.

**Comment by torkelo** [comment]
@DanCech not see any url change in that commit, are you referring to this change on March 20:e? 

https://github.com/grafana/grafana/commit/4d4e165be77ac2824d12b7159cce10cddd543182#diff-552efc423e48c03ad779d46cab991752L429

Where url to grafana.net api was changed from https://grafana.net to https://grafana.com 

The commit you linked to only changed some config key names (while keeping the old keys working for backward compatibility). 

I tested the login with grafana.com oauth and did not find any issue.
